src/test/java/com/puppycrawl/tools/checkstyle/checks/TranslationCheckTest.java

```diff
@@ -78,6 +93,43 @@ public void testTranslation() throws Exception {
             expected);
     }
 
+    /**
+     * Even when we pass several files to AbstractModuleTestSupport#verify,
+     * the check processes it during one run, so we cannot reproduce situation
+     * when TranslationCheck#beginProcessing called several times during single run.
+     * So, we have to use reflection to check this particular case.
+     *
+     * @throws Exception when code tested throws exception
+     */
+    @Test
+    @SuppressWarnings("unchecked")
+    public void testStateIsCleared() throws Exception {
+        final File fileToProcess = new File(getPath("messages_test_de.properties"));
```

Comment:
I am starting to finish up an issue that re-organized our inputs.
Please rename new input files `InputTranslationCheck...`. If file can't be renamed, please give us a reason why.

Reply:
@Nimfadora , please , as we need to change this PR for other reason.